.github/workflows/multibuild.yaml

```diff
@@ -2,6 +2,12 @@ name: Multibuild
 
 on:
   workflow_dispatch:
+    inputs:
+      x64_only:
+        description: "Run x64 builds only"
+        required: true
+        default: false
+        type: boolean
```

Comment:
Added this to speed up the ability to confirm/test x64 releases, since non-x64 releases can take up to 10x longer